src/serialize.h

```diff
@@ -152,12 +152,11 @@ inline float ser_uint32_to_float(uint32_t y)
 // i.e. anything that supports .read(char*, size_t) and .write(char*, size_t)
 //
 
-enum
+enum SerializeType
```

Comment:
However, an enum should always have type int, as long as the number of elements fits in an int, so I don't think that's necessary.